modules/apps/social/social-activity-test/src/testIntegration/java/com/liferay/social/activity/change/tracking/test/SocialRelationTableReferenceDefinitionTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.social.activity.change.tracking.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.change.tracking.test.util.BaseTableReferenceDefinitionTestCase;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.model.change.tracking.CTModel;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.test.util.UserTestUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.social.kernel.service.SocialRelationLocalService;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Brooke Dalton
+ */
+@RunWith(Arquillian.class)
+public class SocialRelationTableReferenceDefinitionTest
+	extends BaseTableReferenceDefinitionTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_user = UserTestUtil.addUser();
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _socialRelationLocalService.addRelation(
+			_user.getUserId(), TestPropsValues.getUserId(), 7);
```

Comment:
Use `SocialRelationConstants` to pass in the type so others reading the code can understand what the value means.